mk/k3d.mk

```diff
@@ -49,6 +49,28 @@ k3d/network/create:
 
 .PHONY: k3d/start
 k3d/start: ${KIND_KUBECONFIG_DIR} k3d/network/create
+	@echo "PORT_PREFIX=$(PORT_PREFIX)"
+	@KUBECONFIG=$(KIND_KUBECONFIG) \
+		k3d cluster create "$(KIND_CLUSTER_NAME)" \
+			-i rancher/k3s:$(CI_K3S_VERSION) \
+			--k3s-arg '--no-deploy=traefik@server:0' \
+			--k3s-arg '--flannel-backend=none@server:*' \
+			--volume "/tmp/calico.yaml:/var/lib/rancher/k3s/server/manifests/calico.yaml" \
```

Comment:
This would normally be "$(TOP)/test/k3d/calico.yaml" but my home path has `@` in it and it triggers this [bug](https://github.com/k3d-io/k3d/issues/506). Will probably work on CI and we would have to live with a guide on how to run it locally.